src/xGadgetron/cGadgetron/include/sirf/Gadgetron/gadget_lib.h

```diff
@@ -356,7 +356,29 @@ namespace sirf {
 		}
 	};
 
-    class GenericReconCartesianFFTGadget : public Gadget {
+	class FFTGadget : public Gadget {
```

Comment:
add some doxygen what this is supposed to do

Reply:
done